scripts/gen_relocate_app.py

```diff
@@ -256,27 +251,20 @@ def generate_linker_script(linker_file, sram_data_linker_file,
             sorted(complete_list_of_sections.items()):
 
         if memory_type != "SRAM":
-            gen_string += MPU_RO_REGION_START.format(memory_type.lower(),
-                                                          memory_type.upper())
-        gen_string += string_create_helper("text",
-                memory_type, full_list_of_sections, 1)
-        gen_string += string_create_helper("rodata",
-                memory_type, full_list_of_sections, 1)
+            gen_string += MPU_RO_REGION_START.format(memory_type.lower(), memory_type.upper())
+        gen_string += string_create_helper("text", memory_type, full_list_of_sections, 1)
+        gen_string += string_create_helper("rodata", memory_type, full_list_of_sections, 1)
         if memory_type != "SRAM":
             gen_string += MPU_RO_REGION_END.format(memory_type.lower())
 
         if memory_type == 'SRAM':
-            gen_string_sram_data += string_create_helper("data",
-                    memory_type, full_list_of_sections, 1)
-            gen_string_sram_bss += string_create_helper("bss",
-                    memory_type, full_list_of_sections, 0)
+            gen_string_sram_data += string_create_helper("data", memory_type, full_list_of_sections, 1)
+            gen_string_sram_bss += string_create_helper("bss", memory_type, full_list_of_sections, 0)
         else:
-            gen_string += string_create_helper("data",
-                    memory_type, full_list_of_sections, 1)
-            gen_string += string_create_helper("bss",
-                    memory_type, full_list_of_sections, 0)
+            gen_string += string_create_helper("data", memory_type, full_list_of_sections, 1)
```

Comment:
I'm not sure having these lines extend beyond 80 columns is an improvement.

Reply:
Is the 80 column limit for .py as well? I did't see it enforced in this file, but I can fix that too if it is specified.